README.md

```diff
@@ -41,8 +41,13 @@ Placeholders are delimited by '-'. Since they're applied with a replace, errors
  - ```uptime``` - Server uptime in ticks
  - ```player_count``` - Number of players on the server
  - ```perm:<permission>``` - 1.0 if the player has the permission, 0.0 otherwise. e.g. ```-perm:essentials.home-```
- - ```aacvl:<check>``` - AAC violation level of the given check (internal name). e.g. ```-aacvl:speed-```
+ - ```aacvl:<check>``` - AAC 1-4 violation level of the given check (internal name). e.g. ```-aacvl:speed-```
  - ```chance:<percentage>%``` - Will be 1.0 percentage% of the time. e.g. ```-chance:34.5%-```
+ - ```cooldown:<arbitrarykey>#<seconds>``` - Returns 0 if ```<arbitrarykey>``` was not seen before by ```/cc cooldown <arbitrarykey>``` command.
+   Otherwise returns time in seconds until last execution of the cooldown command for the given ```<arbitrarykey>``` is more than ```<seconds>``` ago. Never returns a value < 0.
```

Comment:
Maybe:

Returns `0` if `/cc cooldown` has never been executed for the given arbitrarykey, or if it was last executed more than `<seconds>` ago. Otherwise, returns time in seconds since the last execution of the cooldown command.